lib/util.js

```diff
@@ -1627,6 +1644,30 @@ util.decode64 = function(input) {
   return output;
 };
 
+/**
+ * Decodes input according to the "Base64url Encoding" format as specified
+ * in JSON Web Signature (JWS) RFC7517. A URL safe character set is used and
+ * trailing '=', line breaks, whitespace, and other characters are omitted.
+ *
+ * @param input the data to decode.
+ *
+ * @return the decoded value.
```

Comment:
Can we say that the output is a binary string?

Reply:
Sure. Fixing.